Tests/FunctionalTests/FunctionalTests.xcodeproj/xcshareddata/xcschemes/EarlGreyFunctionalTests.xcscheme

```diff
@@ -10,7 +10,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "NO">
+      shouldUseLaunchSchemeArgsEnv = "NO"
+      enableAddressSanitizer = "YES">
```

Comment:
We could have asan turned on for the Objective-C tests and off for the Swift ones, that way we could have both areas covered.